.travis.yml

```diff
@@ -2,7 +2,7 @@
 before_install:
  - sudo add-apt-repository -y ppa:codegear/release
  - sudo apt-get -qq update
- - sudo apt-get install premake4
+ - sudo apt-get install premake4 uuid-dev
```

Comment:
I think this might lead back into that 'package'/'dependency' discussion raised on the forum...